elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "NoRedInk/noredink-ui",
     "summary": "UI Widgets we use at NRI",
     "license": "BSD-3-Clause",
-    "version": "16.2.0",
+    "version": "16.3.0",
```

Comment:
(16.3.0 is already released now)

Reply:
I approved, but this should be updated too.

Reply:
yep, rebased and rebumped